students/laura_denney/lesson09/mailroom_objectoriented.py

```diff
@@ -0,0 +1,280 @@
+#-------------------------------------------------#
+# Title: Object Oriented Mail Room
+# Dev:   LDenney
+# Date:  December 13, 2018
+# ChangeLog: (Who, When, What)
+#   Laura Denney, 10/11/18, Created MailRoom Part 1 File
+#   Laura Denney, 10/12/18, Modified MailRoom Part 1 File
+#   Laura Denney, 10/12/18, Started Part 2
+#   Laura Denney, 10/12/18, Finished Part 2
+#   Laura Denney, 10/31/18, Started MailRoom Part 3
+#   Laura Denney, 11/7/18, Started MailRoom Part 4
+#   Laura Denney, 12/7/18, Started Object Oriented Mail Room
+#   Laura Denney, 12/17/18, Added Saving / Loading Functionality
+#-------------------------------------------------#
+
+#adding date to txt file
+from datetime import date
+
+#file handling help
+import os
+
+#Donor Class contains name, donations
+class Donor(object):
+    name = ""
+    donations = None
```

Comment:
these should not be class variables; class variables are shared among all class instances which is not what you want.

name and donations should be instance attributes.